packages/select/src/js/react.js

```diff
@@ -0,0 +1,63 @@
+/*! [replace-name] v[replace-version] */
+/***************************************************************************************************************************************************************
+ *
+ * select function
+ *
+ * The form select element for multiple options
+ *
+ **************************************************************************************************************************************************************/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+/**
+ * DEFAULT
+ * The select component
+ *
+ * @param  {array}    options  - The options for the select, format: { value: '', text: '' }
+ * @param  {string}   id       - The ID for the select for the label
+ * @param  {string}   name     - The name attribute
+ * @param  {string}   value    - An optional pre-selected value, needs onChange
+ * @param  {function} onChange - A function to be called on change
+ * @param  {string}   block    - The block option
+ */
+const Select = ({ options, id, name, value, onChange, block }) => {
+
+	const attributeOptions = {};
+
+	if( typeof id !== 'undefined' ) {
```

Comment:
it's a good idea to make id required for all form controls to push an accessibility mind set